apis_configs/fence_settings.py

```diff
@@ -135,3 +135,4 @@ def get_from_dict(dictionary, key, default=''):
 DEFAULT_LOGIN_URL_REDIRECT_PARAM = 'redirect'
 
 INDEXD = 'http://indexd-service/'
+ARBORIST = 'http://arborist-service/'
```

Comment:
Can fence default this value if it's not defined?
Otherwise we need to delete and recreate the fence-secret with latest cloud-automation to roll out the new fence - which is annoying ...

Reply:
Also - going forward - I recommended to Ben that `fence` just look for values in `local_settings.creds` or `local_settings.data` - which get sucked in from json blobs, so just:
```
ARBORIST = local_settings.data.get('arborist') or 'fallback value' bla bla
```

Reply:
yep this sounds good. should be easy enough to do it this way instead

Reply:
fence already does it sort of like this:

https://github.com/uc-cdis/fence/blob/300dd25a9d652ef379ccce7870bee6f95270863e/fence/__init__.py#L96-L100

is this what you mean?